README.md

```diff
@@ -12,6 +12,13 @@ Do you want to help testing pre-releases of OSM Surveyor?
 
 ![Screenshot of the map that displays annotations](screenshot.png?raw=true)
 
+# Getting started with development
+
+1. Install [Bundler][5]: `gem install bundler`
+2. Install the Gems (such as Cocoapods and fastlane): `bundle install`
```

Comment:
2. CD to project folder. 
3. Install the Gems (such as Cocoapods and fastlane): `bundle install`. 'bundle install' will install the Gems needed from the existing Gemfile

Reply:
Thanks for the feedback! You are correct: People who are not familiar with the Terminal might forget that second step, cd'ing into the project folder. I've added a paragraph above the list to point that out.